scripts/gh-comments.py

```
@@ -0,0 +1,177 @@
+import subprocess
+import sys
+
+
+def get_pr_number() -> int:
+    """Get PR number from argument or current branch."""
+    if len(sys.argv) > 1:
+        return int(sys.argv[1])
+
+    # Try to get from current branch
+    result = subprocess.run(
+        ["gh", "pr", "view", "--json", "number", "--jq", ".number"],
```

Ruff (0.14.7)

Starting a process with a partial executable path

(S607)